training.rst

```diff
@@ -0,0 +1,113 @@
+Prepare a training on OMERO
+===========================
+
+This chapter describes a setup of a training OMERO.server at 
+your institution. Further, it gives an overview of the other training
+resources as well as hints for trainers about how to present OMERO during a training session.
+
+Resources
+---------
+
+- Community contributions and setups: `Github issue <https://github.com/ome/omero-guides/issues/107>`_.
+
+- `Useful scripts <https://github.com/ome/training-scripts>`_ for setting up the training server.
+
+
+Setup of scripts and OMERO.cli environment
+------------------------------------------
+
+- This guide assumes that you have installed an OMERO.server for training purposes.
+
+- Clone the `training-scripts <https://github.com/ome/training-scripts>`_ repository::
+
+    $ git clone https://github.com/ome/training-scripts.git
+
+- Set up a OMERO.cli as specified under `CLI installation <https://docs.openmicroscopy.org/omero/latest/users/cli/installation.html>`_. Typically, this environment will be used on your local machine. Alternatively, you can use the OMERO.cli environment of the OMERO.server.
+
+Setup of groups and users
+-------------------------
+
+Prepare groups and users as listed in the `provided template sheet <https://github.com/ome/training-scripts/blob/master/maintenance/scripts/create_groups_users_setup>`_. Assuming you have the `template sheet <https://github.com/ome/training-scripts/blob/master/maintenance/scripts/create_groups_users_setup>`_ in the same directory as the script `create_groups_users.sh <https://github.com/ome/training-scripts/blob/master/maintenance/scripts/create_groups_users.sh>`_, you can run::
+
+    $ HOST=$YOUR_SERVER_ADDRESS PASSWORD=$PASSWORD_FOR_ROOT bash create_groups_users.sh
```

Comment:
Could add a `$ cd training-scripts/maintenance/scripts` line before this, especially since users reading the docs won't see the full URLs of the scripts you mention.

Reply:
fixed in https://github.com/ome/omero-guides/pull/236/commits/68522283f4402ea8334b450b705ed8a77c180fc1